### ISSUE TITLE
support jaws feedback

### ISSUE THREAD

**Issue by craigbrett17** [craigbrett17/Blindspot#31]
Originally raised by @kaveinthran as part of issue #24.

**Comment by craigbrett17** [comment]
@kaveinthran: What precisely about Jaws feedback doesn't work?

Admittedly, I haven't tried with JFW for a while, but I believed it worked, at least as of old versions.
